core/src/test/java/org/elasticsearch/index/query/TemplateQueryBuilderTests.java

```diff
@@ -80,7 +80,8 @@ protected void assertBoost(TemplateQueryBuilder queryBuilder, Query query) throw
     @Override
     public void testUnknownField() throws IOException {
         TemplateQueryBuilder testQuery = createTestQueryBuilder();
-        String queryAsString = testQuery.toString().replace("inline", "bogusField");
+        String testQueryAsString = toXContent(testQuery, randomFrom(XContentType.JSON, XContentType.YAML)).string();
```

Comment:
Heads up when you merge with master, I just merged another test where the original test query is modified assuming it is json, so that will need the same treatment as you do here I thinkg: https://github.com/elastic/elasticsearch/pull/14255/files#diff-9dc314365d49d84bff0645c2f9dfd7adR356 (and Overwrites in HasChild/HasParentQueryBuilderTests)